modules/giapi/src/main/scala/giapi/client/package.scala

```diff
@@ -90,12 +90,12 @@ package client {
       *
       * This decision may be review in the future
       */
-    def command(command: Command): F[CommandResult]
+    def command(command: Command, timeout: FiniteDuration): F[CommandResult]
 
     /**
       * Returns a stream of values for the status item.
       */
-    def stream[A: ItemGetter](statusItem: String, ec: ExecutionContext): F[Stream[F, A]]
+    def stream[A: ItemGetter](statusItem: String): F[Stream[F, A]]
```

Comment:
Let's remove the `ExecutionContext` and pass it to the `GiapiConnection` builder